juniper_official/Server-Monitoring/kube-metrics/check-deployment-status-condition.rule

```diff
@@ -0,0 +1,66 @@
+healthbot {
+    topic kube-metrics {
+        rule check-deployment-status-condition {
+            keys [ condition deployment namespace ];
+            synopsis "";
+            description "Checks kube metrics deployment status condition";
```

Comment:
Checks that there are zero unavailable daemons sets and that critical pieces such as docker-registry/docker-registry are up and running.

Reply:
Added description.